src/renderer/dx/DxRenderer.cpp

```diff
@@ -837,7 +834,7 @@ CATCH_RETURN();
     }
     case SwapChainMode::ForComposition:
     {
-        return _sizeTarget.scale(til::math::ceiling, _scale);
+        return _sizeTarget;

```

Comment:
Double-check this.

Reply:
Yep, removing scaling is correct. And I found a bug in resize while I was at it.